community/README.md

```diff
@@ -17,7 +17,7 @@ The Build community abides by the [Contributor Convenant](/Code-of-Conduct.md).
 
 ## Slack
 
-To request an invite to the Slack team, please email build@rubrik.com from the email address to which you would like the request sent. 
+To request an invite to the Slack team, click [here](http://bit.ly/build-slack) the email address to which you would like the request sent. The invite token is `build`.
```

Comment:
Include "and enter" before the "email address."
Should read: ...click here and enter the email address...

Reply:
Fixed.